conf/multiconfig/launch_bitbake_multi_arm64.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -
+
+export MULTICONFIG="qemuarm64"
```

Comment:
Why export? Is OE using it somewhere else and just setting it isn't enough?

Reply:
yes and not, because with multiconfig you affect a directory to the build and the cache are shared between each config. In this case you can change the list the config to use and potentially switch from arm32 to arm64.
I have use the export just for facilitate the reading the command because on sed it's not easy to see the config.